phpstan.neon

```
@@ -4,3 +4,6 @@ parameters:
 
     paths:
         - src/
+
+    ignoreErrors:
```

Why are you adding this?
This shouldn't be needed, you should check the code to update where needed instead of ignoring the error

---

I have an error with PHPStan. We want class-string but, PHPStan want string. How I can solve?

---

You need to check the code to change the places where the instances property is used to add the property in the PHPDOC as class-string. Taking a quick look the first place is in the setInstance method. After that check if any other changes are needed.